embrace-android-sdk/src/integrationTest/kotlin/io/embrace/android/embracesdk/testcases/EmbraceInternalInterfaceTest.kt

```diff
@@ -191,11 +191,11 @@ internal class EmbraceInternalInterfaceTest {
             assertAction = {
                 val session = getSingleSessionEnvelope()
                 val tapBreadcrumb = session.findSessionSpan().findEventOfType(EmbType.Ux.Tap)
-                tapBreadcrumb.attributes?.assertMatches {
-                    "view.name" to "button"
-                    "tap.coords" to "10,99"
```

Comment:
Why did this change? Because some test fixtures changed?

Reply:
View taps no longer capture coordinates by default: https://github.com/embrace-io/embrace-android-sdk/blob/main/UPGRADING.md#upgrading-from-6x-to-7x

#1691 meant the change in behavior wasn't caught in the test. This change will disappear from the PR after a rebase